_posts/en/newsletters/2020-10-14-newsletter.md

```diff
@@ -0,0 +1,179 @@
+  about witness asymmetric payment channels (see [Newsletter
+  #113][news113 witasym]).  The idea behind this proposal is to change
+  how LN users get the evidence they need to prove their channel
+  counterparty cheated.  Currently the evidence is placed in separate
+  transaction for each channel participant; Fournier proposes placing
+  the evidence in a separate signatures ("witnesses") for each
+  participant by using [signature adaptors][topic adaptor signatures].
+  Advantages of this approach are that it provides a protocol that
+  should be easier to integrate with hoped-for improvements to Bitcoin
+  (e.g. [schnorr signatures][topic schnorr signatures] with and without
+  [MuSig][topic musig] aggregation) and hoped-for improvements to LN
+  (e.g. a switch from [HTLCs][topic htlc] to [PTLCs][topic ptlc]).  The approach
+  may also be conceptually simpler, which could help attract more
+  security review of LN's basic operations.
+
+    This week, Fournier linked to an [updated version][fournier v2] of
+    his proposed protocol.  The main difference from the original
+    version is that a party who broadcasts a revoked signature reveals
+    the primary private key they use in the channel's multisig spends.
+    The other party to the channel can use that key combined with their
+    own key to immediately claim all funds in the channel.  Along
+    with another change, <!-- I think it also needs something like
+    Russell's shachain; emailed Fournier 2020-10-11 to confirm --> this
+    should allow storing an entire channel's penalty data in a small
+    number of bytes.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+FIXME:jnewbery or jonatack
+
+{% include functions/details-list.md
+  q0="FIXME"
+  a0="FIXME"
+  a0link="https://bitcoincore.reviews/19339#l-FIXME"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME
+
+- [HWI 1.2.0][] is a new release that adds support for a new hardware
+  device (the BitBox02) and contains multiple bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19954][] Complete the BIP155 implementation and upgrade to TORv3  FIXME:adamjonas
+
+- [Eclair #1537][] extends the `sendtoroute` API call to allow for specifying a
```

Comment:
Maybe this is a British -vs- American English thing, but Jonas and Carl have both used 'allow for' in a way that isn't its primary definition and sounds odd to me. See https://github.com/bitcoinops/bitcoinops.github.io/pull/417#discussion_r436984760. I think "allows a list of channel IDs to be specified with the `--shortChannelIds` flag when calling the `sendtoroute` API" or similar would be clearer.